schemas/com.shopify/update-order/jsonschema/1-0-0

```diff
@@ -0,0 +1,1550 @@
+    },
+    "note_attributes" : {
+      "type" : "array",
+      "items" : { }
+    },
+    "total_discounts" : {
+      "type" : "string"
+    },
+    "billing_address" : {
+      "type" : "object",
+      "properties" : {
+        "first_name" : {
+          "type" : "string"
+        },
+        "city" : {
+          "type" : "string"
+        },
+        "name" : {
+          "type" : "string"
+        },
+        "latitude" : {
+          "type" : [ "number", "null" ],
+          "minimum" : 0
+        },
+        "zip" : {
+          "type" : "string"
+        },
+        "country" : {
+          "type" : "string"
+        },
+        "address2" : {
+          "type" : [ "string", "null" ]
+        },
+        "longitude" : {
+          "type" : [ "number", "null" ],
+          "minimum" : 0
+        },
+        "company" : {
+          "type" : [ "string", "null" ]
+        },
+        "country_code" : {
+          "type" : "string",
+          "enum" : [ "AZ", "NF", "PW", "HN", "CY", "GI", "AI", "LC", "UG", "LS", "FM", "LI", "MA", "KI", "TM", "MO", "MM", "TZ", "CR", "LV", "MX", "ME", "PY", "TF", "ID", "IR", "SY", "BF", "BM", "VU", "GH", "JP", "TH", "CX", "GU", "GP", "NI", "NP", "CN", "BG", "MR", "AR", "PG", "IE", "EE", "ZW", "UY", "RU", "CG", "TR", "BO", "QA", "MP", "GM", "BB", "AW", "AN", "GQ", "ST", "LA", "DJ", "AO", "KN", "BT", "GT", "TN", "AS", "TL", "CF", "SD", "BS", "LR", "TO", "IM", "PH", "BY", "AX", "MC", "SR", "UM", "PF", "AQ", "LB", "EG", "TJ", "MU", "SO", "VC", "NU", "SC", "AC", "GD", "AL", "HR", "GG", "ML", "SN", "PK", "SJ", "BJ", "KY", "EC", "BN", "MV", "BA", "CK", "FI", "IO", "MH", "TK", "NZ", "NE", "KZ", "ES", "KW", "BI", "HM", "FK", "GF", "BD", "PR", "UZ", "TC", "MS", "IS", "GW", "TG", "KG", "MG", "WS", "NL", "CD", "BV", "GR", "PA", "SZ", "FO", "VI", "TW", "MW", "JO", "PL", "GA", "TT", "DE", "KP", "CV", "KR", "GS", "DO", "WF", "TD", "SG", "CH", "HK", "BZ", "RO", "AD", "ZM", "IL", "DK", "PN", "GB", "VA", "TV", "SL", "BE", "HT", "MN", "VE", "CC", "GN", "IT", "CU", "VG", "TA", "HU", "FR", "CO", "SA", "CZ", "UA", "US", "DZ", "ZA", "JM", "SI", "DM", "PT", "NA", "GE", "NR", "RE", "PM", "KE", "NC", "GL", "MQ", "AE", "PE", "MZ", "NG", "LY", "CA", "MY", "SB", "KM", "MT", "ET", "AM", "FJ", "VN", "SM", "JE", "NO", "OM", "KH", "SV", "AU", "SH", "YT", "ER", "RS", "BR", "LK", "RW", "YE", "BH", "GY", "SE", "LU", "SK", "IQ", "AG", "MD", "CM", "MK", "IN", "CL", "CI", "LT", "BW", "AT" ]
+        },
+        "last_name" : {
+          "type" : "string"
+        },
+        "address1" : {
+          "type" : "string"
+        },
+        "province" : {
+          "type" : "null"
+        },
+        "province_code" : {
+          "type" : "null"
+        },
+        "phone" : {
+          "type" : [ "string", "null" ]
+        }
+      },
+      "additionalProperties" : false
+    },
+    "order_number" : {
+      "type" : "integer",
+      "maximum" : 32767,
+      "minimum" : 0
+    },
+    "email" : {
+      "type" : "string"
```

Comment:
`format: email`

Reply:
Can i tie a format to a field name when calling schema_guru?

Reply:
Not yet unfortunately. Also, we're adding `email` to Igluctl only in upcoming R8 release.